### ISSUE TITLE
getinfo() deprecated in Bitcoin Core 0.16 onwards

### ISSUE THREAD

**Issue by masterquiroga** [BitGo/bitgod#16]
[#11382](https://github.com/bitcoin/bitcoin/issues/11382)
> "WARNING: getinfo is deprecated and will be fully removed in 0.16. Projects should transition to using `getblockchaininfo`, `getnetworkinfo`, and `getwalletinfo` before upgrading to 0.16"

From PR [#8780](https://github.com/bitcoin/bitcoin/pull/8780):
 
>Some of the more serious problems with it are:
> * It combines information from different subsystems (wallet, networking, blockchain, ...), so its implementation is messy by definition and has to lock all over the place
> * It is ill-defined and without focus - what information should be added, what shouldn't
> * It behaves differently with and without wallet (and what should it do with multiple wallets)

**Comment by masterquiroga** [comment]
As a matter of fact, trying this out:
```
$ curl -i     -H "Authorization: Basic ${YOUR_AUTH_STRING_HERE}"     -H "Host: localhost"     -H "Content-Type: application/json"     -H "Accept: application/json"     -X POST -d '{"id":1,"method":"getinfo","params":[],"jsonrpc":"2.0"}' http://localhost:18332
HTTP/1.1 404 Not Found
Content-Type: application/json
Date: Wed, 13 Dec 2017 02:22:55 GMT
Content-Length: 76
```

Results in:
```
{"result":null,"error":{"code":-32601,"message":"Method not found"},"id":1}
```

**Comment by OttoAllmendinger** [comment]
thanks for reporting! fixed in #28
